db/migrate/20220915053651_create_comment.rb

```diff
@@ -0,0 +1,20 @@
+class CreateComment < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :author, null: false, index: true
```

Comment:
- [ ] This should be  `t.references :author, foreign_key: { to_table: 'users' } `, which will create author_id and a corresponding index

Reply:
Hi @GzimAsani , Thanks for the review but, if you see the full file I have performed the `foreign_key` relationship in a different way in the same file
[please refer this lines](https://github.com/RitobrotoMukherjee/ror-blog/pull/1/files/e6e12f25d4e6f5aa5b2082fd84d70cd4d3596360#diff-9664ff84e4fe8d96dd36eb1cae632f501c1273b2b01226fe567296c4e68c8395R13-R15), and it is creating the `author_id` and a corresponding `index` too. You check in your `psql` after running the `rake db:migrate`.
- **The reference to this solution is in official docs [link 1](https://guides.rubyonrails.org/active_record_migrations.html#foreign-keys), and [link 2](https://guides.rubyonrails.org/active_record_migrations.html#changing-the-output-of-running-migrations)**

---
## For easy reference
```
class CreateProducts < ActiveRecord::Migration[7.0]
  def change
    suppress_messages do
      create_table :products do |t|
        t.string :name
        t.text :description
        t.timestamps
      end
    end

    say "Created a table"

    suppress_messages {add_index :products, :name}
    say "and an index!", true

    say_with_time 'Waiting for a while' do
      sleep 10
      250
    end
  end
end
```

Reply:
@RitobrotoMukherjee Yep, your app functionality was perfect, I just thought to mention it since that is the easiest and more understandable way for a starter. But if you already know that, you are good to go just update your readme file

Reply:
@GzimAsani Thanks for the clarification. Yes I know that process too, I have used the same in the migration to create the `likes` table. 😄 And thanks for your time. 💯

Reply:
This change has been done well. Nice implementation I have learnt something new